java/src/jmri/jmrit/beantable/turnout/TurnoutTableDataModel.java

```diff
@@ -382,35 +382,31 @@ public void setValueAt(Object value, int row, int col) {
         if (col == INVERTCOL) {
             if (t.canInvert()) {
                 t.setInverted((Boolean) value);
-                fireTableRowsUpdated(row, row);
```

Comment:
Why is this removed? Is it not needed?
I ask since I don't know and want to understand the code before I approve it.

Reply:
Fair question :-)

An Event Listener has been attached to each Turnout in the super, BeanTableDataModel.updateNameList()
When a Property Change Event is triggered, the Property Name is checked in 
`protected boolean matchPropertyName(java.beans.PropertyChangeEvent e)`

If the Property Name matches, then fireTableRowsUpdated() is called.
This PR updates the list of matched Property Names ( and converts the ifs to switch )
These fireTableRowsUpdated are therefore not required.

It's preferable to fire on the Prop Change Event rather than the setValueAt,
when the Turnout is updated anywhere else in JMRI, ( ie jmrit.beantable.beanedit.TurnoutEditAction ) the Table updates.

If the Property Change requires more than a row update, an override of the standard
`public void propertyChange(java.beans.PropertyChangeEvent e)`
is used.